### ISSUE TITLE
Ships spawning underneath land all over the place

### ISSUE THREAD

**Issue by minyassa** [RustyMods/VikingNPC#5]
We have the raider ships spawning on the water under our land masses and then attacking from below, smashing into objects with their masts, and generally being an annoyance. We've had to use the kill switch to get rid of them as they can sail underneath a barn and shoot livestock from underneath, etc.

**Comment by RustyMods** [comment]
How do you have a land mass above water ?

**Comment by minyassa** [comment]
There's a water layer either over the entire map. If you go no-clip you can
go down through the land and see it. So they're spawning underneath the
land mass there, and trying to sail but the land above gets too low and
they get caught wherever it's lower than the top of the mast. They also
aggro whenever they are underneath anything that's within their spotting
distance, so we're seeing harpoons fly up from underneath the ground. Just
lost all the pigs in my barn from them until I just turned off their
spawning everywhere except the middle of the ocean. So they can't spawn in
the waters near any shores now because they'll also spawn anywhere within
the biome and be at sea level.

Here's a screencap showing the water layer under the land; note the troll
and greydwarf spawner above.

[image: image.png]

**Comment by minyassa** [comment]
No, neither of us touched it--we first noticed the issue in our rented
server, and then I installed it in my private server on a new r2mm profile
to check it and it was set to all biomes. I had to set it to oceans
manually. Could another mod have messed with the config somehow?
